geode-apis-compatible-with-redis/src/main/java/org/apache/geode/redis/internal/executor/BaseSetOptions.java

```diff
@@ -47,6 +48,14 @@ public boolean isXX() {
     return exists.equals(Exists.XX);
   }
 
+  public void enableChange() {
```

Comment:
This might be better named "setCH" and changed to a true setter that takes an argument, for consistency with a "setX/isX" boolean naming convention.